src/panels/lovelace/common/directives/long-press-directive.ts

```diff
@@ -12,6 +12,7 @@ interface LongPress extends HTMLElement {
 }
 interface LongPressElement extends Element {
   longPress?: boolean;
+  hasDblClick?: boolean | undefined;
```

Comment:
We shouldn't do it like this. Instead we should pass it into the directive as option.